code/modules/projectiles/projectile/energy/ebow.dm

```diff
@@ -1,12 +1,24 @@
 /obj/item/projectile/energy/bolt //ebow bolts
 	name = "bolt"
 	icon_state = "cbbolt"
-	damage = 8
+	damage = 15
 	damage_type = TOX
 	nodamage = 0
-	knockdown = 100
-	stutter = 5
-	drowsy = 50
+	stamina = 60
+	eyeblur = 10
+	slur = 5
+	var/hardstun_ds = 0
```

Comment:
![image](https://user-images.githubusercontent.com/2003111/73211477-90519c00-4109-11ea-909c-aa6737e374e5.png)

You can replace ALL of this (so the vars you made and the on_hit) with uhh
knockdown = 160
stamina = 0
knockdown_stamoverride = 60

This applies hardstun of 0.01*160 = 1.6 ds, and to protect from this you would need 50% or above energy armor, and does 60 stamina by overriding the knockdown

Also that would disarm both hands instead of just one, which I feel is better given this is a strong 12 tc weapon (or should be)

The large memebows are probably better off with knockdown=100 or something to make them comparable to oldtaser, with stam as the same value if you want considering they're already.. really strong, even for "illegal" tech.

Also I like the idea of making large ebows conflict with other large ebows because otherwise I can just carry multiple and quickdraw but whatever.

Regardless up to you, your thing *works*, but it's far more elegant (and probably in gameplay because the overrides are.. touchy as well as the stam dmg/hardstun ds being 0) to use builtins unless you're going for a very snowflakey effect.